### ISSUE TITLE
Nvda does not automatically read the progress bar of a chrome download

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#7171]
When downloading in chrome, a chrome download page appears on the chrome download bar, however nvda does not read this chrome bar automatically, or even by pressing the arrow keys or down to read the bar again. To obtain the updated download percentage, you must update the virtual buffer by pressing nvda + f5.Is it possible for this bar to be read automatically, or is it possible to read its updated content by pressing the arrows?To play:Start a file download in chrome, preferably large to give time to do the tests.With the download in progress press ctrl + j.Chrome will open the downloads page.Navigate with the arrow keys until you find the bar of progress.Wait a few seconds and see navigating with the arrows that nvda will read the same download percentage. Also the new values ​​of the progress bar are not read automatically.The expected would be that the nvda read the values ​​of the progress bar automatically, or that it was possible to see the updated value by walking with the arrows on the page.Finally press nvda + f5 and navigate with the arrows until the bar of progress, see that the updated value is read by the nvda.

Follow the log:

Input: kb(laptop):downArrow
IO - speech.speak (15:49:23.700):
Speaking [u'barra de progresso', u'66']
IO - inputCore.InputManager.executeGesture (15:49:28.118):
Input: kb(laptop):NVDA+f1
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (15:49:28.447):
Developer info for navigator object:
name: None
role: ROLE_UNKNOWN
states: 
isFocusable: False
hasFocus: False
Python object: <NVDAObjects.Dynamic_ProgressBarIAccessibleWindowNVDAObject object at 0x0A9ADDB0>
Python class mro: (<class 'NVDAObjects.Dynamic_ProgressBarIAccessibleWindowNVDAObject'>, <class 'NVDAObjects.behaviors.ProgressBar'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: (320, 305, 487, 4)
value: None
appModule: <'appModuleHandler' (appName u'chrome', process ID 5588) at address 5e7cd70>
appModule.productName: u'Google Chrome'
appModule.productVersion: u'58.0.3029.96'
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 918840
windowClassName: u'Chrome_RenderWidgetHostHWND'
windowControlID: 297182528
windowStyle: 1445986304
windowThreadID: 3464
windowText: u'Chrome Legacy Window'
displayText: u''
IAccessibleObject: <POINTER(IAccessible) ptr=0x9f500f8 at aa16d00>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=918840, objectID=-4, childID=-145761
IAccessible accName: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible accRole: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible accState: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible accDescription: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible accValue: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))

**Comment by ehollig** [comment]
@jcsteh or @feerrenrut, is this something that would need to be filed with Chrome?

**Comment by Adriani90** [comment]
@fernando-jose-silva does it work in NVDA 2019.1.1 and Chrome 73 now as expected? Or are you still having this issue?

**Comment by fernando-jose-silva** [comment]
perfect, everything works fine now.

**Comment by Adriani90** [comment]
Closing as works for me
